spec/mailers/previews/customer_mailer_preview.rb

```diff
@@ -55,6 +69,13 @@ def recovered_from_link
     CustomerMailer.recovered_from_link(recovered_record)
   end
 
+  def bike_possibly_found_email
+    target = Bike.stolen.last
+    match = Bike.abandoned.last
+    contact = CustomerContact.build_bike_possibly_found_notification(target, match)
```

Comment:
Why do this rather than inspecting the most recent matching customer mailer?

Reason I like the most recent match - it's convenient if the mailer previews match the most recent because then you can go and generate another one of the things, and then view that instance in the mailer preview.

Reply:
Good point thanks! Updated in 23a9d1be